docs/fsharp-cheatsheet.md

```diff
@@ -67,88 +68,88 @@ Other common examples are `F` or `f` for 32-bit floating-point numbers, `M` or `
 
     let s, f, d, bi = 4.14F, 4.14, 0.7833M, 9999I
 
-	// [fsi:val s : float32 = 4.14f]
-	// [fsi:val f : float = 4.14]
-	// [fsi:val d : decimal = 0.7833M]
-	// [fsi:val bi : System.Numerics.BigInteger = 9999]
+    // [fsi:val s : float32 = 4.14f]
+    // [fsi:val f : float = 4.14]
+    // [fsi:val d : decimal = 0.7833M]
+    // [fsi:val bi : System.Numerics.BigInteger = 9999]
 
 See [Literals (MSDN)](http://msdn.microsoft.com/en-us/library/dd233193.aspx) for complete reference.
 
 <a name="Functions"></a>Functions
 ---------
-The `let` keyword also defines named functions.
+The `let` keyword also defines named functions:
 
-	let negate x = x * -1 
-	let square x = x * x 
-	let print x = printfn "The number is: %d" x
+    let negate x = x * -1 
+    let square x = x * x 
+    let print x = printfn "The number is: %d" x
 
     let squareNegateThenPrint x = 
-		print (negate (square x)) 
+        print (negate (square x)) 
 
 ### Pipe and composition operators
-Pipe operator `|>` is used to chain functions and arguments together. Double-backtick identifiers are handy to improve readability especially in unit testing:
+Pipe operator `|>` is used to chain functions and arguments together. Double-backtick identifiers are handy to create a symbol with spaces in it to improve readability, especially for unit testing:
```

Comment:
The note about unit testing seemed like a non sequitur until I decoded it. I think it could read more smoothly. How about “Extra-long identifiers in double-backticks are especially handy for unit test names”.